client/mysqladmin.cc

```diff
@@ -1428,10 +1428,12 @@ static void usage(void)
   version		Get version info from server");
 }
 
+#define SHOW_MSG_LEN (FN_REFLEN + 20)
```

Comment:
Why introduce this #define SHOW_MSG_LEN? It only makes it harder to understand the code, that the right size is being passed to snprintf() (as opposed to just sizeof()), and that the buffer is large enough that it will not be truncated and corrupt the command to be contructed.

Reply:
You are right, I was thinking this existed elsewhere too somehow

Reply:
Actually this was by request to remove the magic values https://github.com/MariaDB/server/pull/3077#discussion_r1496437200